### ISSUE TITLE
Post "https://hub.docker.com/v2/access-tokens/desktop-generate": EOF

### ISSUE THREAD

**Issue by wuxins** [docker/for-mac#6862]
while trying desktop login, following error occurs:
Post "https://hub.docker.com/v2/access-tokens/desktop-generate": EOF

[https://user-images.githubusercontent.com/40260824/239660629-f17efb73-538b-45f3-8ff4-a113c8f30224.png](url)

**Comment by icemaple** [comment]
me 2

**Comment by bsousaa** [comment]
It was probably a transient issue. Does it keep happening?

**Comment by yiguotang** [comment]
I think it's a network problem. When I turn on the global vpn, Then login successfully.

**Comment by wuxins** [comment]
With VPN connected , show my operation below :

<img width="977" alt="image" src="https://github.com/docker/for-mac/assets/40260824/0c5edf51-d11f-4fc6-b51d-07393524736a">
<img width="1286" alt="image" src="https://github.com/docker/for-mac/assets/40260824/5563c446-b115-4d60-9f54-127aea38e737">
<img width="874" alt="image" src="https://github.com/docker/for-mac/assets/40260824/717a7cec-6906-4a2c-8e39-9f3a431eb2fc">
<img width="852" alt="image" src="https://github.com/docker/for-mac/assets/40260824/e04d8c98-7ed6-4bc3-8d86-ef1c0b4ef3e8">

With VPN closed , I can't open the website auth page below 
<img width="1758" alt="image" src="https://github.com/docker/for-mac/assets/40260824/0099fac4-763f-495a-946b-8058b928c8c8">

**Comment by wuxins** [comment]
Diagnostic ID:
    64DB07F8-599F-4916-8EEF-8993542AC181/20230610064209

Function is ok at the first time I used it 。
Currently I search & pull images via command line ～

**Comment by linkerlau** [comment]
> I think it's a network problem. When I turn on the global vpn, Then login successfully.

you are right, i try to resolve it many times, it works fine now, thank you!

**Comment by 5thgfka** [comment]
vpn set to global mode is not work for me. :(

**Comment by luckygopher** [comment]
me too

**Comment by rogerliu2019** [comment]
same here, I'v been suffering from it for really a long time even if I enabled VPN from Singapore or South Africa. Anyone help?

**Comment by siburuxue** [comment]
login success!
<img width="1270" alt="image" src="https://github.com/docker/for-mac/assets/15135780/14a944f5-77cc-41ee-b9cf-7362c4fa3a2f">

**Comment by Jalr4ever** [comment]
Yeah, seems like a vpn proxy issue, while you must **keep your browser proxy same as your docker desktop.**

In my scenario, I use `127.0.0.1:1087` as proxy in Chrome SwitchyOmega, when I configured this proxy in docker desktop, it was fixed.

**Comment by myhongkongzhen** [comment]
@Jalr4ever, may I know how to check the proxy port from Chrome browse? would you please share some ideas? Many thanks!

**Comment by ASanlittleboss** [comment]
turn off the manual option, it's work
![image](https://github.com/docker/for-mac/assets/49724125/3e034e57-4411-46c3-949c-a1f373aa2580)

**Comment by panlina** [comment]
Still an issue in Chinese. Global VPN does not solve the issue. Currently I change back to 4.20 and login from terminal and it succeeds, though the desktop still shows not logged in.
